src/cmdstan/command.hpp

```diff
@@ -255,6 +257,11 @@ int command(int argc, const char *argv[]) {
   }
   std::shared_ptr<stan::io::var_context> init_context = get_var_context(init);
 
+  // Read clamped parameters file
+  std::string clamped_params_filename
+      = dynamic_cast<string_argument *>(parser.arg("clamped_params"))->value();
+  std::shared_ptr<stan::io::var_context> clamped_params_context = get_var_context(clamped_params_filename);
```

Comment:
Do all other references to this pointer in stan use a smart pointer?

Reply:
I think this is the only pointer. Everything else is a reference to the value here.

Reply:
Then shouldn't a unique_ptr be better?

Reply:
Oh, you're right it's a simpler way to solve the same problem. I'll change these to unique ptrs later if I mess with this pull any.

Reply:
I tried unique_ptrs and they worked but then got scared and switched back cause we're returning them from functions and was worried the destructor might get called or something.